### ISSUE TITLE
Keyboard shortcut requests

### ISSUE THREAD

**Issue by darnn** [SubtitleEdit/subtitleedit#2584]
Hey, this isn't urgent or anything, but when you get around to it, could you add keyboard shortcuts to the following, please?

File:
* Close original subtitle
* Import plain text
* Import time codes
* Export -> plain text
Tools:
* Make new empty translation from subtitle file
Video:
* Open
* Close
Synchronization:
* Point sync via other subtitle

Thanks!

**Comment by darnn** [comment]
Thank you! Could you update the beta?

**Comment by niksedk** [comment]
Beta updated: https://github.com/SubtitleEdit/subtitleedit/releases/download/3.5.4/SubtitleEditBeta.zip

**Comment by darnn** [comment]
Thanks!!

**Comment by niksedk** [comment]
Updated beta: https://github.com/SubtitleEdit/subtitleedit/releases/download/3.5.4/SubtitleEditBeta.zip
Better?

**Comment by darnn** [comment]
Yeah, I've actually tested them all now. They all work, but "close original subtitle" still isn't saved in the settings file. Also, if you're still making changes, and if it isn't too hard, I'd still like a keyboard shortcut for file->export->plain text.
And, as long as we're here, I seem to have discovered that the key combination alt+return, which I set to video->fullscreen, works when English or Russian is the currently selected language (that is, in the language bar, the one I'm typing in), but doesn't work when the language is Hebrew, and I press right-alt+return. It does work with left-alt+return, though.
